src/components/Staking/index.tsx

```diff
@@ -1,5 +1,6 @@
 import { Contract } from 'near-api-js/lib/contract'
 import { Box, Container, SimpleGrid } from '@chakra-ui/react'
+import IdoStakingForm from './IdoStakingForm'
```

Comment:
What are the differences between _IdoStakingForm_ & _StakingForm_?

Reply:
IdoStakingForm clone from StakingForm with new UI.
Will remove StakingForm when completely switch over.

.